docs/how/search.md

```diff
@@ -39,6 +39,41 @@ The filters sidebar sits on the left hand side of search results, and lets users
  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/filters_highlighted.png" />
 </p>
 
+### Advanced Filters
```

Comment:
Thank you so much for updating these docs!

Reply:
of course :)